api/v1beta1/common_types.go

```diff
@@ -218,27 +218,38 @@ const (
 
 	// MachineSetPreflightCheckKubeadmVersionSkew is the name of the preflight check
 	// that verifies if the machine being created or remediated for the MachineSet conforms to the kubeadm version
-	// skew policy that requires the machine to be at the same version as the control plane.
+	// skew policy that requires the machine to be at the same minor version as the control plane.
 	// Note: This is a stopgap while the root cause of the problem is fixed in kubeadm; this check will become
 	// a no-op when this check will be available in kubeadm, and then eventually be dropped when all the
-	// supported Kuberenetes/kubeadm versions have implemented the fix.
+	// supported Kubernetes/kubeadm versions have implemented the fix.
```

Comment:
Can we add a link to the upstream issue that should be fixing this?

Reply:
@fabriziopandini do you have a link?

Reply:
I'm not sure there is an issue (@neolit123 might know), and TBH I think  we should probably drop this comment.

The underlying problem is that from a kubeadm minor to another it might change how kubeadm represent the cluster configuration, and older releases trying to join might fail (and making older releases "future proof" is not simple)

Reply:
If someone has a link I'm happy to add it via this PR, otherwise I think this is pretty orthogonal to this PR

Reply:
ACK, up to you if to keep the comment or drop it

Reply:
Hm yeah it's misleading. I dropped it